### DISCUSSION TITLE
How can I see the album art of a song?

### DISCUSSION THREAD

**Discussion by halloleo** [losnoco/Cog#328] [Q&A] [answered]
How can I see the album art of a song in Cog? All songs appear just as a text line to me:

![Screen Shot 2022-09-28 at 3 21 42 pm](https://user-images.githubusercontent.com/1521852/192694194-7cf5fcf3-b278-4ceb-9df5-7d535785ce01.png)

**Comment by kode54** [comment, marked as answer]
The info dialog. The info button in the toolbar. That is all. There is no album art in the playlist.

**Reply by halloleo** [reply]
Ok. Thanks.
